tests/Mado/QueryBundle/Objects/QueryBuilderFactoryTest.php

```diff
@@ -83,7 +83,7 @@ public function testFilterWithListTypeOnEmbeddedField()
     {
         $queryBuilderFactory = new QueryBuilderFactory($this->manager);
         $queryBuilderFactory->setFields([ 'id' ]);
-        $queryBuilderFactory->setAndFilters([ '_embedded.group.id|list' => '42, 33' ]);
+        $queryBuilderFactory->setAndFilters([ '_embedded.group.id|list' => '42, 33', '_embedded.group.company.name|contains' => 's' ]);
```

Comment:
Old test is no more useful? Why did you update test instead of create new one? Old test have wrong behavior?

Reply:
Because I had a problem on construct join so in this test i wanna be sure that the join is done only one time